src/plugins/intel_cpu/src/nodes/deconv.cpp

```diff
@@ -484,39 +412,40 @@ void Deconvolution::getSupportedDescriptors() {
 }
 
 void Deconvolution::initPaddingR(const Shape &inShape, const Shape &outShape) {
-    for (size_t i = 0; i < paddingR.size(); i++) {
+    for (size_t i = 0; i < deconvAttrs.paddingR.size(); i++) {
         int with_group = getAlgorithm() == Algorithm::DeconvolutionGrouped ? 1 : 0;
         const auto& weightDims = getWeightDims();
         int krn = weightDims[with_group + 2 + i];
         int src = outShape.getStaticDims()[2 + i];
         int dst = inShape.getStaticDims()[2 + i];
 
-        krn = (krn - 1)*(dilation[i] + 1) + 1;
-        paddingR[i] = (dst - 1) * stride[i] - (src - krn + paddingL[i]);
+        krn = (krn - 1)*(deconvAttrs.dilation[i] + 1) + 1;
+        deconvAttrs.paddingR[i] = (dst - 1) * deconvAttrs.stride[i] - (src - krn + deconvAttrs.paddingL[i]);
     }
 }
 
 void Deconvolution::setPostOps(dnnl::primitive_attr& attr, const VectorDims& dims) {
     dnnl::post_ops ops;
 
     // ONEDNN define the convolution forward as :
-    //  [N, OC, OH, OW] = [N, IC, IH, IW]* [OC, IC, KH, KW]
+    //  [N, deconvAttrs.OC, OH, OW] = [N, deconvAttrs.IC, IH, IW]* [deconvAttrs.OC, deconvAttrs.IC, KH, KW]
```

Comment:
please remove deconvAttrs prefix such type of comments
These comments explain the general logic, and not the exact data types we have